skyvern/forge/agent.py

```diff
@@ -376,19 +378,71 @@ async def execute_step(
                 url=e.url,
                 error_message=e.error_message,
             )
-            task = await self.update_task(
+            failure_reason = f"Failed to navigate to URL. URL:{e.url}, Error:{e.error_message}"
+            try:
+                await self.fail_task(task, None, failure_reason)
+            except RetryError:
+                LOG.error(
+                    "Failed to update status and failure reason in database. Task might going to be time_out",
+                    task_id=task.task_id,
+                    step_id=step.step_id,
+                )
+            finally:
+                await self.send_task_response(
```

Comment:
I don't think we should send_task_response again here since this is the exception handler for `send_task_response` already. we might get into an infinite loop

Reply:
Em. There must be something wrong wih the github diff. I didn't do `send_task_response` again in the `FailedToSendWebhook`..

<img width="625" alt="Code 2024-06-12 10 20 57" src="https://github.com/Skyvern-AI/skyvern/assets/26927551/41ef2d31-fc70-4b05-8f3d-7c323824780b">